packaging/rpm/scripts/after_install_host.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+echo "This software may collect information about you and your use of the software, and send that to Microsoft."
+echo "Please visit http://aka.ms/dotnet-cli-eula for more information."
+
+# Run 'dotnet new' as the user to trigger the first time experience to initialize the cache
+echo "Welcome to .NET Core!
```

Comment:
I agree we need to fix this. We should separate the condition of first run and telemetry and show the telemetry message in the first run despite the fact that the cache has been expanded. I believe we have a bug tracking this already, but I also believe this should not block the PR.